src/main/scala/io/iohk/ethereum/db/storage/AppStateStorage.scala

```diff
@@ -14,6 +14,8 @@ class AppStateStorage(val dataSource: DataSource) extends TransactionalKeyValueS
 
   val namespace: IndexedSeq[Byte] = Namespaces.AppStateNamespace
   def keySerializer: Key => IndexedSeq[Byte] = _.getBytes
+
+  def keyDeserializer: IndexedSeq[Byte] => Key = k => new String(k.toArray)
```

Comment:
I'd consider specifying charset here and 2 lines below. Otherwise hard-to-track issues may arise e.g. when moving data between different OS

Reply:
i was thinking about it, but decided to not touch it for now. But you are probably right, this pr break state eitherway so i will do it here.